be/src/types/bitmap_value.cpp

```diff
@@ -862,9 +862,8 @@ void BitmapValue::to_array(std::vector<int64_t>* array) const {
         array->emplace_back(_sv);
         break;
     case BITMAP: {
-        for (unsigned long ptr_value : *_bitmap) {
-            array->emplace_back(ptr_value);
-        }
+        (*array).resize(_bitmap->cardinality());
```

Comment:
I'm wondering whether it will be faster if you use raw_vector::make_room, it will eliminate the memset operation.

Reply:
After testing, it is not a performance bottleneck in the current scenario, but better to use make_room instead

Reply:
Fixed